README.rst

```diff
@@ -87,6 +87,8 @@ or with ``--help`` as so::
     Password for 'username' in 'system':
     $ keyring get system username
     password
+    $ keyring list app_name
```

Comment:
We should think carefully about the "command" here. It's called "list", but "list" could mean "list users" or "list systems" or "list users and systems". I wonder if we shouldn't instead allow a generic "list" that takes a string and matches any system:password where that string is found. Let's take some time to think about other use-cases other than exact matches of a system.